scanners/trivy/parser/__testFiles__/juice-shop-v12.10.2-empty-results.json

```diff
@@ -0,0 +1,180 @@
+        },
+        {
+          "created": "2021-10-12T21:23:22.113753293Z",
+          "created_by": "USER 1001",
+          "comment": "buildkit.dockerfile.v0",
+          "empty_layer": true
+        },
+        {
+          "created": "2021-10-12T21:23:22.113753293Z",
+          "created_by": "EXPOSE map[3000/tcp:{}]",
+          "comment": "buildkit.dockerfile.v0",
+          "empty_layer": true
+        },
+        {
+          "created": "2021-10-12T21:23:22.113753293Z",
+          "created_by": "CMD [\"npm\" \"start\"]",
+          "comment": "buildkit.dockerfile.v0",
+          "empty_layer": true
+        }
+      ],
+      "os": "linux",
+      "rootfs": {
+        "type": "layers",
+        "diff_ids": [
+          "sha256:39982b2a789afc156fff00c707d0ff1c6ab4af8f1666a8df4787714059ce24e7",
+          "sha256:f8700d3a252fffe60e30bc672e8a6560f30a3ce8816f2ad396020553fe4d9210",
+          "sha256:b8f0e895f5208b04d533d013ddec6f12642fdd679ef70bc1497ffe733c97428b",
+          "sha256:446ec7c50f08cfba388bcebe29f54b2a46a5ddccdabd6b4caac21cbdb7c60b4b",
+          "sha256:14bfcbbb53f34ede6cd7b031e162c5943a738eb21543ca1fdfdc0cc1ab578c07",
+          "sha256:96fd22f85d18e17255224d3bf9a75ea7da9985ecad19780e762815410c64a780",
+          "sha256:882c984cafed0fa20487f0b4f95474af7d46cff44d735d266633ebcea37e2bd1",
+          "sha256:873323b172036f6876ecc04895b0f8832ccecef9601ae723633fe08b9886ea83"
+        ]
+      },
+      "config": {
+        "Cmd": [
+          "npm",
+          "start"
+        ],
+        "Entrypoint": [
+          "docker-entrypoint.sh"
+        ],
+        "Env": [
+          "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+          "NODE_VERSION=12.22.6",
+          "YARN_VERSION=1.22.5"
+        ],
+        "Labels": {
+          "maintainer": "Bjoern Kimminich \u003cbjoern.kimminich@owasp.org\u003e",
+          "org.opencontainers.image.authors": "Bjoern Kimminich \u003cbjoern.kimminich@owasp.org\u003e",
+          "org.opencontainers.image.created": "”2021-10-12T21:09:21Z”",
+          "org.opencontainers.image.description": "Probably the most modern and sophisticated insecure web application",
+          "org.opencontainers.image.documentation": "https://help.owasp-juice.shop",
+          "org.opencontainers.image.licenses": "MIT",
+          "org.opencontainers.image.revision": "3d8a93e",
+          "org.opencontainers.image.source": "https://github.com/juice-shop/juice-shop",
+          "org.opencontainers.image.title": "OWASP Juice Shop",
+          "org.opencontainers.image.url": "https://owasp-juice.shop",
+          "org.opencontainers.image.vendor": "Open Web Application Security Project",
+          "org.opencontainers.image.version": "12.10.2"
+        },
+        "User": "1001",
+        "WorkingDir": "/juice-shop",
+        "ExposedPorts": {
+          "3000/tcp": {}
+        },
+        "ArgsEscaped": true
+      }
+    }
+  },
+  "Results": []
```

Comment:
This won't affect the new check btw. Because [] is an array.

Reply:
@c0olix Thanks for the review. This made me realize that the trivy test for empty results never actually ran correctly ^_^". This PR should fix the issue, https://github.com/secureCodeBox/secureCodeBox/pull/1787 .